MatrixSDK/Data/MXRoomLastMessage.m

```diff
@@ -58,6 +59,27 @@ - (instancetype)initWithEvent:(MXEvent *)event
     return self;
 }
 
+- (nullable NSData*)sensitiveData;
+{
+    NSError* error;
+    NSData* archived = [NSKeyedArchiver archivedDataWithRootObject:[self sensitiveDataDictionary]
+                                             requiringSecureCoding:NO
+                                                             error:&error];
+    
+    if (error) {
+        MXLogDebug(@"[MXRoomLastMessage] did fail to archive sensitiveDataDictionary. Error: %@", error.description);
```

Comment:
For future reference: errors are best logged via `MXLogErrorDetails` which will also send them to Sentry (if enabled), or even `MXLogFailureDetails` if we do not expect this error to ever occur